src/pages/CommandView/form-data/get-validator/get-any-validators.ts

```diff
@@ -0,0 +1,94 @@
+import { FormValidation } from '@rjsf/core'
```

Comment:
As mentioned above, the `Any` parameter types is nothing but trouble. The legacy version of the UI essentially forked the form library into a whole new npm package in order to add in these *pythonic* types, adding hundreds of lines of impenetrable code in the process.

This file provides a field validator for `Any` parameters. Hopefully it's easier to follow than it's predecessor.